charts/scalardb-cluster/README.md

```diff
@@ -25,7 +25,6 @@ Current chart version is `1.0.0-SNAPSHOT`
 | fullnameOverride | string | `""` | String to fully override scalardb-cluster.fullname template |
 | nameOverride | string | `""` | String to partially override scalardb-cluster.fullname template (will maintain the release name) |
 | scalardbCluster.affinity | object | `{}` | The affinity/anti-affinity feature, greatly expands the types of constraints you can express. |
-| scalardbCluster.existingSecret | string | `""` | Name of existing secret to use for storing database username and password. |
```

Comment:
This is an old (deprecated) way to set a username/password. Now, we recommend using `scalardbCluster.scalardbClusterNodeProperties` and `scalardbCluster.secretName` to set the username/password.